openc3-cosmos-init/init.sh

```diff
@@ -101,7 +101,9 @@ if [ "${OPENC3_CLOUD}" == "local" ]; then
     # Create a new user scriptrunner on MinIO use mc admin user.
     mc admin user add openc3minio ${OPENC3_SR_BUCKET_USERNAME} ${OPENC3_SR_BUCKET_PASSWORD} || exit 1
     # Once the user is successfully created you can now apply the getonly policy for this user.
-    mc admin policy attach openc3minio script --user=${OPENC3_SR_BUCKET_USERNAME} || exit 1
+    # "|| true" is required on subsequent startups due to the following error that is thrown:
+    # mc: <ERROR> Unable to make user/group policy association. The specified policy change is already in effect. (Specified policy update has no net effect).
+    mc admin policy attach openc3minio script --user=${OPENC3_SR_BUCKET_USERNAME} || true
```

Comment:
This is a good catch.  Are there any flags to the mc command itself that would have the same effect?

Reply:
I didn't see any flags to adjust this behavior. I did look back into Minio's code, and it appears that this was introduced in this [pull request](https://github.com/minio/minio/pull/16182/files#diff-b36131f5f2912eeb5ef4c601441d61a1f48cc621a01c5ed7cedc9fad2ee3b6b6R920).